packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1852,7 +1852,9 @@ curl -v -X POST \
 
 #### Enroll Custom HOTP Factor
 
-Enrolls a user with a Custom HMAC-based One-Time Password (HOTP) Factor. The enrollment process involves passing a Factor profile ID and shared secret for a particular token.
+Enrolls a user with a Custom HMAC-based One-Time Password (HOTP) Factor. The enrollment process involves passing a `factorProfileId` and `sharedSecret` for a particular token.
+
+A Factor Profile represents a particular configuration of the Custom HOTP factor.  It includes certain properties that match the hardware token end users possess such as the HMAC Algorithm, passcode length, and time interval.  There can be multiple Custom HOTP factor profiles per org, but users can only be enrolled for one Custom HOTP factor total.  Admins can create Custom HOTP factor profiles in the Okta admin UI following the instructions laid out in this [help documentation](https://help.okta.com/en/prod/Content/Topics/Security/mfa-totp-seed.htm) and copy the `factorProfileId` from the UI into the below API request.
```

Comment:
several edits. Please remove double spacing after each period. We only need one to follow a period.

other changes I have in **bold** - except can't seem to bold the comma that was needed before "such as"

I removed "total", as it's redundant. Saying "can only be enrolled for one" means "total"

We refer to the Admin UI as Admin Console

Also - we use alias links when linking to the Product Help doc. I have provided that alias. It will be live no later than EOD Monday. https://help.okta.com/en/prod/okta_help_CSH.htm#ext-mfa-totp

Removed "below" and replaced with "following" for accessibility reasons

A Factor Profile represents a particular configuration of the Custom HOTP factor. It includes certain properties that match the hardware token **that** end users possess, such as the HMAC Algorithm, passcode length, and time interval. There can be multiple Custom HOTP factor profiles per org, but users can only be enrolled for one Custom HOTP factor. Admins can create Custom HOTP factor profiles in the Okta **Admin Console** following the instructions on the **[Custom TOTP Factor help page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-mfa-totp)**. Then, copy the `factorProfileId` from the **Admin Console** into the **following** API request: